### PR TITLE
[v1.0.1-rhel] Backport "Drop vergen dependency"

### PR DESCRIPTION
**Pull request by Luap99**
Backport commit e824402c85d7.

We only need the version, build time, rust target triple, and git commit
for the version command. These values can easily be generated with little
code so we do not need an extra dependency for this.

The main reason for this change is that vergen will fail if no git
commit is found. On distro build systems this is often the case since
they build from a source tar without the git repo. Instead of erroring
we should just show an empty commit in the version output.

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM
/lgtm

**Comment by TomSweeneyRedHat** [comment]
Hold this until approval from RHEL, then we'll need to create a new release.
But then again, I'm not seeing this vendored into the Podman v4.0-rhel branch?  How does that work with Rust into Golang?

**Comment by mheon** [comment]
Aardvark is a separate project, Podman is not involved

**Pull request merged**
